.github/workflows/test_suite.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       matrix:
         browser: [chrome, firefox]
-        python-version: [pypy3, 3.6, 3.7, 3.8]
```

Comment:
@RonnyPfannschmidt I removed pypy3 here because it doesn't pass the check for `python_requires` in setuptools, any thoughts on supporting pypy here?

Reply:
@mshriver latest pypy supports python 3.7 - so either we go back to supporting that that or we have to drop

Reply:
Where does/did the `pypy3` support come from, are you aware of any RH teams using it?  I'd say drop, but that's ignorant of current uses.